blocks/large-promo-block/README.md

```diff
@@ -114,28 +121,14 @@ export default {
 - `content.websites[arcSite].website_section`
 - `content.promo_items`
 
-## Internationalization fields
+## Internationalization
 
-| Phrase key | Default (English)     |
-| ---------- | --------------------- |
-| `key`      | `english translation` |
+n/a
```

Comment:
```suggestion
global.and-text
global.by-text
global.sponsored-content
```

Reply:
I don't know that we need to include all translations here.  I think the keys should be enough?  @matthewroach @JackHowa

Reply:
I think keys are a good start. Adding the default would be better though. If you look at the product-assortment-carousel-block, there's a good example of this for the dynamic variables. Overall, since this block does not actually have the intl.json, it should be straight-forward. It's using the global-phrases block